src/Umbraco.Infrastructure/Migrations/Upgrade/V_15_0_0/ConvertLocalLinks.cs

```diff
@@ -57,6 +66,10 @@ public ConvertLocalLinks(
         _linkMigrationTracker = linkMigrationTracker;
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ConvertLocalLinks"/> class.
+    /// </summary>
+    [Obsolete("Please use the constructor taking all parameters. Scheduled for removal in Umbraco 18.")]
```

Comment:
Okay this is very nitpicky so feel free to ignore. I mean yes we could remove this in 18, but every migration prior to 17 will be removed in the next major version 🤷